kratos/tests/auxiliar_files_for_python_unittest/mdpa_files/test_assign_mpcs_to_neighbours_process.mdpa

```diff
@@ -0,0 +1,220 @@
+   25   0.8000000000   0.0000000000   0.0000000000
+   26   0.7166666667   0.2883974596   0.0000000000
+   27   0.6000000000   0.5000000000   0.0000000000
+   28   0.6000000000   0.1535898385   0.0000000000
+   29   0.6000000000   0.0000000000   0.0000000000
+   30   0.5000000000   0.3267949192   0.0000000000
+   31   0.4447777778   0.1590523136   0.0000000000
+   32   0.4000000000   0.5000000000   0.0000000000
+   33   0.4000000000   0.0000000000   0.0000000000
+   34   0.3000000000   0.3267949192   0.0000000000
+   35   0.2686666667   0.1471342048   0.0000000000
+   36   0.2000000000   0.5000000000   0.0000000000
+   37   0.2000000000   0.0000000000   0.0000000000
+   38   0.0000000000   0.5000000000   0.0000000000
+   39   0.0000000000   0.2500000000   0.0000000000
+   40   0.0000000000   0.0000000000   0.0000000000
+End Nodes
+
+
+Begin Elements Element2D3N// GUI group identifier: Domain
+        1          0    40    37    39 
+        2          0    25    23    21 
+        3          0    32    36    34 
+        4          0    34    36    39 
+        5          0    32    34    30 
+        6          0    30    34    31 
+        7          0    32    30    27 
+        8          0    31    34    35 
+        9          0    30    31    28 
+       10          0    27    30    26 
+       11          0    35    34    39 
+       12          0    28    31    29 
+       13          0    28    29    25 
+       14          0    26    30    28 
+       15          0    28    25    26 
+       16          0    26    25    21 
+       17          0    33    29    31 
+       18          0    33    31    35 
+       19          0    24    27    26 
+       20          0    24    26    21 
+       21          0    37    33    35 
+       22          0    37    35    39 
+       23          0    19    24    21 
+       24          0    36    38    39 
+       25          0    22    17    20 
+       26          0     9     7     3 
+       27          0    11    15    13 
+       28          0    14    12    10 
+       29          0     2     5     4 
+       30          0    17    14    16 
+       31          0     1     2     3 
+       32          0    15    18    20 
+       33          0    12     9     6 
+       34          0     5    11     8 
+       35          0    13    15    20 
+       36          0     4     5     8 
+       37          0     4    10     6 
+       38          0     4     6     3 
+       39          0     6     9     3 
+       40          0    10    12     6 
+       41          0    10     4     8 
+       42          0    10     8    13 
+       43          0    13     8    11 
+       44          0    16    14    10 
+       45          0    16    10    13 
+       46          0    16    13    20 
+       47          0     2     4     3 
+       48          0    17    16    20 
+End Elements
+
+Begin SubModelPart GENERIC_Domain // Group Domain // Subtree GENERIC
```

Comment:
If this submodelpart is not used you can simply removed to make the mdpa even smaller. I understand you only need master/slave ones

Reply:
Since some application require the MPCs to be applied to an specific model part (e.g. FluidDynamics -> fluid_computing_model_part). The process need the name of this model part. In this case, GENERIC_Domain plays this role.